Article/Article.js

```diff
@@ -3,18 +3,18 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
```

Comment:
A "this" can be used here before the domElement on the right of the equal sign.
*this.expandButton = this.domElement.querySelector('.expandButton');